### PR TITLE
[associative.reqmts] Turn emphasis into a note.

### LINKED ISSUE #606: Words in italics
Italic words are only specified to be the terms or syntactic categories. What about other kind of usage, i.e. to emphasize?

For example, in [associative.reqmts]/4:

> The phrase “equivalence of keys” means the equivalence relation imposed by the comparison and _not_ the `operator==` on keys.

Here the word "not" is italicized. Is this intended?

BTW, for this particular half of the sentence, is it better to be a note or a footnote? Note that the `operator==` can mean equivalence relation on keys occasionally.

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #606.

### CODE REVIEW

File: source/containers.tex [hunk 1]
```diff
@@ -1507,11 +1507,7 @@
 
 \pnum
 The phrase ``equivalence of keys'' means the equivalence relation imposed by the
-comparison and
-\textit{not}
-the
-\tcode{operator==}
-on keys.
+comparison.
```
Comment (jwakely) [inline source/containers.tex]: Would it be a further improvement to say "imposed by the comparison object" not just "imposed by the comparison"?

File: source/containers.tex [hunk 2]
```diff
@@ -1520,6 +1516,10 @@
 comparison object
 \tcode{comp},
 \tcode{comp(k1, k2) == false \&\& comp(k2, k1) == false}.
+\begin{note}
+This is not necessarily the same as the result of the \tcode{operator==}
+on keys.
```
Comment (jwakely) [inline source/containers.tex]: Maybe "This is not necessarily the same as the result of `k1 == k2`." ?

File: source/containers.tex (hunk 1 above)
Comment (jensmaurer) [reply source/containers.tex]: Fixed.

File: source/containers.tex (hunk 2 above)
Comment (jensmaurer) [reply source/containers.tex]: Fixed.

**Comment by jensmaurer** [comment]
Updates per @jwakely's comments.

**Pull request merged**